### DISCUSSION TITLE
Who owns the copyright to the generated data?

### DISCUSSION THREAD

**Discussion by a000744210** [synthetichealth/synthea#1167] [Q&A]
If I used data/schema generated from this tool that someone else generated, are they legally allowed to sue me for copyright infringement?

**Comment by jawalonoski** [comment]
The software is released under Apache 2.0 according to our [LICENSE](https://github.com/synthetichealth/synthea/blob/master/LICENSE).

The resulting synthetic data is free from cost, privacy, and security restrictions.

Some of the exported data formats that have their own license. For example, our default export format HL7 FHIR has its own license: https://www.hl7.org/fhir/license.html

The exported data uses terminologies that have their own license. See our [NOTICE](https://github.com/synthetichealth/synthea/blob/master/NOTICE).

If you have a legal question you should consult a lawyer.

I am going to lock this conversation because this is not a forum for legal consultation.
